### PR TITLE
Dark mode and responsiveness.

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
This PR has no major feature development and is only refactored code with responsiveness and dark mode toggle added.

Live  : https://dark-mode-and-responsiveness.netlify.app/

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Looking good to me

File: styles/rules.css [hunk 1]
```diff
@@ -1,5 +1,19 @@
 .rules-container{
```
Comment (hsnice16) [inline styles/rules.css:1]: Great, nice naming convention.

File: styles/index.css [hunk 2]
```diff
@@ -42,12 +47,118 @@ nav{
...
+        box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+        z-index: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        display: none;
+
+    }
+    .nav-link {
```
Comment (hsnice16) [inline styles/index.css:120]: You can give a line space here too. As you have given at other places.

**Review by hsnice16 — APPROVED** [review]
Great work, loved the UI and dark mode. Nice. Just added some suggestions.

File: styles/rules.css (hunk 1 above)
Comment (samyukthasudhakar) [reply styles/rules.css:1]: Thank you :D

File: styles/index.css (hunk 2 above)
Comment (samyukthasudhakar) [reply styles/index.css:120]: Sure will do.

**Pull request merged**